pkg/deployer/sched/sched.go

```diff
@@ -49,16 +48,9 @@ func Deploy(log tlog.Logger, opts Options) error {
 		return err
 	}
 
-	rteMf, err := rtemanifests.GetManifests(opts.Platform, "")
```

Comment:
Where do we get manifests for the scheduler?

Reply:
line 58. This was to configure the scheduler plugin to look in the RTE namespace when looking for the NRT objects.

Reply:
I see, thanks!